### PR TITLE
Translate activities.

### PR DESCRIPTION
**Pull request by mlvernay**
Use modeltranslation to translate Activite name. At this point only the name has to be translated.

Once deployed 

```py
from erp.models import Activite

for fr, en in (
    ("Apiculteur", "Beekeeper"),
    ("Bien-être", "Wellness"),
    ("Bijouterie joaillerie", "Jewelry shop"),
    ("Blanchisserie teinturerie", "Laundry and dry cleaning"),
    ("Boucherie / commerce de viande", "Butchery / meat trade"),
    ("Boulangerie", "Bakery"),
    ("Boulangerie Pâtisserie", "Bakery / pastry shop"),
    ("Boulodrome", "Bowling alley"),
    ("Bowling", "Bowling"),
    ("Bricolage, matériaux, travaux", "Do-it-yourself, materials, work"),
    ("Casino", "Casino"),
    ("Chenil, fourrière", "Kennel, pound"),
    ("Commission conciliation", "Conciliation Commission"),
    ("École élémentaire", "Elementary school"),
    ("Emploi, formation", "Employment, training"),
    ("Encadreur enlumineur", "Framer illuminator"),
    ("Accessoires", "Accessories"),
    ("Accompagnement personnes agees", "Accompanying the elderly"),
    ("Achat or", "Gold purchase"),
    ("Aéroport", "Airport"),
    ("Agence de voyage", "Travel agency"),
    ("Arts de la table", "Tableware"),
    ("Centre de ressources et d'information", "Resource and information center"),
    ("Chambre de commerce et d'industrie", "Chamber of Commerce and Industry"),
    ("Cimetière", "Cemetery"),
    ("École primaire (regroupement maternelle et élémentaire)", "Elementary school (kindergarten and elementary)"),
    ("Épicerie", "Grocery store"),
    ("Établissement de santé", "Health care facility"),
    ("Établissement militaire", "Military establishment"),
    ("Imprimerie photocopie reliure", "Printing, photocopying and binding"),
    ("Location articles loisirs et sport", "Rental of leisure and sport articles"),
    ("Maison de services au public", "House of services to the public"),
    ("Agence événementielle", "Event agency"),
    ("Accrobranche", "Accrobranche"),
    ("Agence postale", "Postal agency"),
    ("Aide sociale à l'enfance : action éducative", "Child welfare : educational action"),
    ("Aménagement maison : cuisine salle de bain salon", "Home furnishing : kitchen, bathroom, living room"),
    ("Boutique de mariage", "Wedding store"),
    ("Cycle vente et entretien", "Sales and maintenance cycle"),
    ("EHPAD", "EHPAD"),
    ("Friperie", "Thrift shop"),
    ("Hôtel", "Hotel"),
    ("Hypnothérapeute", "Hypnotherapist"),
    ("Jeux vidéo", "Video games"),
    ("Notaire", "Notary"),
    ("Opéra", "Opera"),
    ("Produits de terroir", "Local products"),
    ("Salle de danse", "Dance hall"),
    ("Salle de spectacle", "Showroom"),
    ("Aide à la personne", "Help for the person"),
    ("Anesthésie", "Anesthesia"),
    ("Art", "Art"),
    ("Assurance", "Insurance"),
    ("Athlétisme", "Athletics"),
    ("Audio prothésiste", "Audio prosthetist"),
    ("Auto école", "Auto school"),
    ("Commerce bio", "Organic trade"),
    ("Dermatologie vénéréologie", "Dermatology venereology"),
    ("Gare avec desserte train à grande vitesse (TAGV)", "Train station with high speed train service (TAGV)"),
    ("Gastro-entérologie hépatologie", "Gastroenterology and hepatology"),
    ("Magasin de cbd", "CDB store"),
    ("Musée", "Museum"),
    ("Ophtalmologie", "Ophthalmology"),
    ("Barbier", "Barber shop"),
    ("Bar tabac", "Tobacco bar"),
    ("Brocante", "Brocante"),
    ("Bureau de poste", "Post office"),
    ("Café, bar, brasserie", "Coffee, bar, brasserie"),
    ("Cafés et thés", "Coffee and tea shops"),
    ("Camping caravaning", "Camping and caravanning"),
    ("Cantine", "Canteen"),
    ("Cardiologie", "Cardiology"),
    ("Carrosserie", "Body shop"),
    ("Concessionnaire automobile", "Car dealership"),
    ("Confiserie", "Confectionery"),
    (
        "Direction départementale de l'emploi, du travail, des solidarités et de la protection des populations",
        "Departmental directorate of employment, labor, solidarity and population protection",
    ),
    ("Électricien", "Electrician"),
    ("Instruments et matériel de musique", "Musical instruments and equipment"),
    ("Matériel de pêche", "Fishing equipment"),
    ("Ordre des avocats", "Bar Association"),
    ("Salles de remise en forme", "Fitness centers"),
    ("Sécurité sociale, mutuelle santé", "Social security, mutual health insurance"),
    ("Agence de travail temporaire", "Temporary employment agency"),
    ("Agence immobilière", "Real estate agency"),
    ("Ambulances", "Ambulances"),
    ("Animalerie", "Pet shop"),
    ("Antiquaire", "Antique shop"),
    ("Aquarium", "Aquarium"),
    ("Architecte", "Architect"),
    ("Arène", "Arena"),
    ("Armurerie coutellerie", "Armoury and cutlery"),
    ("Artisanat", "Handicraft"),
    ("Association", "Association"),
    ("Auditorium et salle de conférence", "Auditorium and conference room"),
    (
        "Autres établissements pour adultes et familles en difficulté",
        "Other facilities for adults and families in difficulty",
    ),
    ("Avocat", "Lawyer"),
    ("Biscuiterie", "Cookie factory"),
    ("Construction de maisons, immeubles", "Construction of houses, buildings"),
    ("Équipements du foyer", "Home furnishings"),
    ("Guichet france services", "Guichet france services"),
    ("Institut de formation", "Training institute"),
    ("Magasin de tissus", "Fabric store"),
    ("Motocycle vente et entretien", "Motorcycle sales and maintenance"),
    ("Organisation patronale, professionnelle, syndicale", "Employer, professional and trade union organizations"),
    ("Orthopédie", "Orthopedics"),
    ("Orthoptie", "Orthoptics"),
    ("Parc des expositions", "Exhibition centre"),
    ("Parfumerie beauté", "Perfumery and beauty"),
    ("Parking & stationnement", "Parking & parking"),
    ("Patinoire", "Skating rink"),
    ("Pâtisserie", "Pastry shop"),
    ("Pédiatrie", "Pediatrics"),
    ("Pédicure-podologue", "Pedicure-podologue"),
    ("Pépinière", "Nursery"),
    ("Agence départementale d'information sur le logement", "Departmental agency for information on housing"),
    ("Chapeaux et couvre-chefs", "Hats and headgear"),
    ("Château", "Castle"),
    ("Chirurgien dentiste", "Dental surgeon"),
    ("Décoration Design", "Design decoration"),
    ("Disquaire", "Record store"),
    ("Distribution spécialisée / commerce de gros", "Specialized distribution / wholesale"),
    ("Électroménager et matériel audio-vidéo", "Household appliances and audio-video equipment"),
    ("Ergothérapeute", "Occupational therapist"),
    ("Fruits et légumes", "Fruits and vegetables"),
    ("Médecin généraliste", "General practitioner"),
    ("Menuiserie, ébénisterie", "Carpentry, cabinet making"),
    ("Office du tourisme", "Tourist office"),
    ("Pharmacie", "Pharmacy"),
    ("Piscine", "Swimming pool"),
    ("Pompes funèbres", "Funeral parlour"),
    ("Salle de concert", "Concert hall"),
    ("Centre de gestion de la fonction publique territoriale", "Management center of the local civil service"),
    ("Chambres d'hôtes, gîte, pension", "Bed and breakfast, guest house"),
    ("Chocolatier", "Chocolatier"),
    ("Cigarette électronique", "Electronic cigarette"),
    ("Cinéma", "Cinema"),
    ("Circuit sportif deux roues / voiture", "Sports circuit two wheels / car"),
    ("Clinique", "Clinic"),
    ("Coiffure", "Hairdresser"),
    ("Collège", "College"),
    ("Commerce automobile", "Car trade"),
    ("Commissariat de Police", "Police station"),
    ("Contrôle technique auto", "Technical control car"),
    ("Cordonnerie serrurerie", "Shoe repair shop"),
    ("Crèche", "Crèche"),
    ("Autre", "Other"),
    ("Crèmerie Fromagerie", "Creamery Cheese factory"),
    ("Cure thermale", "Spa treatment"),
    ("Agence matrimoniale", "Matrimonial agency"),
    ("Comptable expert-comptable", "Accountant"),
    ("École maternelle", "Nursery school"),
    ("Endocrinologue", "Endocrinologist"),
    ("Epicerie fine", "Delicatessen"),
    ("Etablissement pénitentiaire", "Correctional facility"),
    ("Orthodontie", "Orthodontics"),
    ("Oto-rhino-laryngologie", "Otolaryngology"),
    ("Papeterie, presse, journaux", "Stationery, press, newspapers"),
    ("Parc", "Park"),
    ("Parc d’attraction", "Amusement park"),
    ("Photographie", "Photography"),
    ("Plateaux et terrains de jeux extérieurs", "Outdoor playgrounds and playgrounds"),
    ("Point justice", "Point of justice"),
    ("Poterie verrerie céramique", "Pottery, glassware, ceramics"),
    ("Produits surgelés", "Frozen products"),
    ("Transport", "Transport"),
    ("Afpa", "Afpa"),
    ("Association aide aux victimes", "Victims' aid association"),
    ("Cyclisme", "Cycling"),
    ("Gare routière", "Bus station"),
    ("Hôpital", "Hospital"),
    ("Information Touristique", "Tourist information"),
    ("Jeux jouets", "Toys and games"),
    ("Loisirs créatifs", "Creative hobbies"),
    ("Ostéopathie", "Osteopathy"),
    ("Personnes âgées : foyer restaurant", "Elderly people : foyer restaurant"),
    ("Personnes âgées : hébergement", "Elderly people : accommodation"),
    ("Point conseil budget", "Budget advice point"),
    ("Profession libérale", "Liberal profession"),
    ("Psychologie, Psychiatrie", "Psychology, Psychiatry"),
    ("Psychomotricien", "Psychomotrician"),
    ("Ressourcerie", "Resource centre"),
    ("Restaurant scolaire", "School restaurant"),
    ("Restauration rapide", "Fast food restaurant"),
    ("Revêtements murs et sols", "Wall and floor coverings"),
    ("Chambre agriculture", "Chamber of agriculture"),
    ("Conservatoire et école de musique", "Conservatory and music school"),
    ("Coworking", "Coworking"),
    ("Déchetterie", "Waste disposal center"),
    (
        "Direction départementale de la protection des populations",
        "Departmental directorate for the protection of populations",
    ),
    ("Karaoké", "Karaoke"),
    ("Mercerie", "Haberdashery"),
    (
        "Mission locale pour l'insertion professionnelle et sociale des jeunes",
        "Local mission for the professional and social integration of young people",
    ),
    ("Monument historique", "Historical monument"),
    ("Neurologie", "Neurology"),
    ("Opticien", "Optician"),
    ("Orthophonie", "Speech therapy"),
    ("Plomberie, chauffage", "Plumbing, heating"),
    ("Pneumologie", "Pneumology"),
    (
        "Poissonnerie / commerce de poissons, crustacés et mollusques",
        "Fishmonger's / fish, crustacean and mollusc trade",
    ),
    ("Port", "Port"),
    ("Pressing, nettoyage", "Pressing, cleaning"),
    ("Primeur", "Primeur"),
    ("Résidence, foyer", "Residence, home"),
    ("Retraite", "Retirement"),
    ("Rhumatologie", "Rheumatology"),
    ("Centre d'examen", "Examination center"),
    ("Centre social", "Social center"),
    ("Chambre metier", "Chamber of Commerce"),
    ("Coopérative agricole", "Agricultural cooperative"),
    ("Fleuriste", "Florist"),
    ("Galerie d'art", "Art gallery"),
    ("Garage automobile", "Car garage"),
    ("Gare sans desserte train à grande vitesse (TAGV)", "Railway station without high speed train service (TAGV)"),
    ("Gendarmerie", "Gendarmerie"),
    ("Géomètre", "Surveyor"),
    ("Glacier", "Glacier"),
    ("Gymnase", "Gymnasium"),
    ("Gynécologie", "Gynaecology"),
    ("Hématologie", "Hematology"),
    ("Herboristerie naturopathie", "Herbalist naturopathy"),
    ("Horlogerie", "Watchmaking"),
    ("Hôtel restaurant", "Hotel restaurant"),
    ("Huissier", "Bailiff"),
    ("Administration publique", "Public administration"),
    ("Commerce", "Business"),
    ("Espace collaboratif", "Collaborative space"),
    ("Hypermarché", "Hypermarket"),
    ("Infirmier", "Nurse"),
    ("Informatique", "Computer science"),
    ("Jardin botanique et/ou zoologique", "Botanical and/or zoological garden"),
    ("Jardinerie", "Garden center"),
    ("Kiosque (théâtre, pizza, journaux)", "Kiosk (theater, pizza, newspapers)"),
    ("Laboratoire d'analyse médicale", "Medical analysis laboratory"),
    ("Laverie", "Laundromat"),
    ("Librairie", "Bookstore"),
    ("Lieu de culte", "Place of worship"),
    ("Lieu de visite", "Place of visit"),
    ("Lingerie sous-vêtements", "Lingerie and underwear"),
    (
        "Association de gestion du fonds pour l'insertion professionnelle des personnes handicapées",
        "Association for the management of the fund for the professional integration of disabled people",
    ),
    ("Equipement sportif", "Sports equipment"),
    ("Literie", "Bedding"),
    ("Location de matériels", "Material rental"),
    ("Location véhicules", "Vehicle rental"),
    ("Luminaire", "Lighting"),
    ("Lycée", "High School"),
    ("Magasin de bois de chauffage", "Firewood store"),
    ("Mairie", "City Hall"),
    ("Maison de santé ou centre de santé", "Health house or health center"),
    ("Marché", "Market"),
    ("Maroquinerie sellerie articles de voyage", "Leather goods, saddlery, travel goods"),
    ("Massages", "Massages"),
    ("Masseur kinésithérapeute", "Masseur kinésithérapeute"),
    ("Médecine du travail", "Occupational medicine"),
    ("Numérique", "Digital"),
    ("Plage", "Beach"),
    ("Banques, caisses d'épargne", "Banks, savings banks"),
    ("Bâtiment d'accueil", "Reception building"),
    ("Bazar", "Bazaar"),
    ("Bibliothèque médiathèque", "Library / media library"),
    ("Caviste / commerce de détail de boissons", "Liquor store / beverage retailer"),
    ("Centre commercial", "Shopping center"),
    ("Centre culturel", "Cultural center"),
    ("Centre de loisirs", "Leisure center"),
    ("Centre de vacances", "Vacation center"),
    ("Centre de vaccination", "Vaccination center"),
    ("Centre équestre", "Equestrian center"),
    ("Centre médical", "Medical center"),
    ("Centre religieux", "Religious center"),
    ("Charcuterie", "Delicatessen"),
    ("Chirurgie", "Surgery"),
    ("Salle spécialisée", "Specialized room"),
    ("Centre d'information sur les droits des femmes et des familles", "Information center on women and family rights"),
    ("Chaussures", "Shoes"),
    ("Collectivité territoriale", "Territorial community"),
    ("Communauté de communes", "Community of municipalities"),
    ("Courtier", "Broker"),
    ("Droguerie", "Drugstore"),
    ("Immobilier", "Real estate"),
    ("Musique", "Music"),
    ("Point accueil numerique", "Digital reception point"),
    ("Salle de combat", "Fighting room"),
    ("Salle de jeux", "Games room"),
    ("Salle de réception", "Reception room"),
    ("Salle des fêtes", "Party room"),
    ("Salle non spécialisée", "Non-specialized room"),
    ("Services techniques", "Technical services"),
    ("Sex shop", "Sex shop"),
    ("Sports et loisirs", "Sports and leisure"),
    ("Sports nautiques", "Water sports"),
    ("Stade", "Stadium"),
    ("Bureau de douane", "Customs office"),
    ("Déménageur", "Movers"),
    ("Discothèque", "Discotheque"),
    ("Espace numérique", "Digital space"),
    (
        "Fédération départementale pour la pêche et la protection du milieu aquatique",
        "Departmental Federation for fishing and protection of the aquatic environment",
    ),
    ("Greffe des associations", "Registry of associations"),
    ("Point information jeunesse", "Youth information point"),
    ("Puériculture", "Childcare"),
    ("Radiodiagnostic et imagerie médicale", "Radiodiagnosis and medical imaging"),
    ("Rempailleur tapissier chaises fauteuils", "Upholsterer and chair maker"),
    ("Réparation auto et de matériel agricole", "Car and agricultural equipment repair"),
    ("Restaurant", "Restaurant"),
    ("Retouche", "Retouching"),
    ("Sage-femme", "Midwife"),
    ("Salon de thé", "Tearoom"),
    ("Service ou aide à domicile", "Service or home help"),
    ("Station lavage auto", "Car wash"),
    ("Stomatologie", "Stomatology"),
    ("Agence de publicité", "Advertising agency"),
    ("Bailleur social", "Social landlord"),
    ("Centre d’information et d’orientation", "Information and orientation center"),
    ("Garderie", "Daycare"),
    ("Magasin de souvenirs", "Souvenir store"),
    ("Refuge animalier", "Animal shelter"),
    ("Salle multisports", "Multi-sports hall"),
    ("Skatepark", "Skatepark"),
    ("Soins de beauté", "Beauty treatments"),
    ("Sophrologue", "Sophrologist"),
    ("Spa", "Spa"),
    ("Station service", "Service station"),
    ("Supérette", "Supermarket"),
    ("Supermarché", "Supermarket"),
    ("Syndic, gérance immo", "Syndic, property management"),
    ("Tatouage Piercing", "Tattoo Piercing"),
    ("Taxidermie", "Taxidermy"),
    ("Téléphonie", "Telephony"),
    ("Vente de matériel et équipement industriels", "Sale of industrial material and equipment"),
    ("Brûlerie", "Burner shop"),
    ("Ecole de danse", "Dance school"),
    ("Espace vert et naturel", "Green and natural area"),
    ("Meubles ameublement", "Furniture and furnishings"),
    ("Organisme de conseil", "Counseling organization"),
    ("Point d'information local dédié aux personnes âgées", "Local information point dedicated to the elderly"),
    ("Tabac", "Tobacco"),
    ("Taxi", "Cab"),
    ("Tennis", "Tennis"),
    ("Textile hors habillement", "Textile except clothing"),
    ("Thalassothérapie", "Thalassotherapy"),
    ("Tribunal", "Court"),
    ("Centrale nucléaire", "Nuclear power plant"),
    ("Centre de protection maternelle et infantile (pmi)", "Mother and child protection center (pmi)"),
    ("Centre national de la fonction publique territoriale", "National Center for Territorial Public Service"),
    ("Etablissement de prévention", "Prevention establishment"),
    ("Mission d'accueil et d'information des associations", "Mission of reception and information of associations"),
    ("Caisse d'allocations familiales (caf)", "Caisse d'allocations familiales (caf)"),
    ("Maison départementale des personnes handicapées", "Departmental house for disabled people"),
    ("Théâtre", "Theater"),
    ("Toilettes publiques", "Public toilets"),
    ("Toiletteur", "Toilet attendant"),
    ("Traiteur", "Caterer"),
    ("Trésorerie", "Treasury"),
    ("Université ou école supérieure", "University or college"),
    ("Urologie", "Urology"),
    ("Vente à distance", "Distance selling"),
    ("Vente d'articles de sport", "Sale of sporting goods"),
    ("Vente de matériel médical", "Sale of medical equipment"),
    ("Vente de matériel pour piscine", "Pool equipment sales"),
    ("Vêtements", "Clothing"),
    ("Vétérinaire", "Veterinary"),
):
    try:
        act = Activite.objects.get(nom_fr=fr)
        act.nom_en = en
        act.save()
        print(f"{fr} now translated")
    except Activite.DoesNotExist:
        print(f"No activity with name {fr}")


```

### CODE REVIEW

File: bin/post_deploy [hunk 1]
```diff
@@ -1,3 +1,2 @@
 #!/bin/env bash
-
```
Comment (Anto59290) [inline bin/post_deploy:2]: We could revert this change if unnecessary

File: conftest.py [hunk 2]
```diff
@@ -121,7 +121,7 @@ def activite(db):
         "Aide à la personne",
         "Aide sociale à l'enfance : action éducative",
         "Ambulances",
-        "Aménagement maison : cuisine salle de bain salon",
```
Comment (Anto59290) [inline conftest.py:124]: Is it a breaking space vs non breaking space change ?

**Review by Anto59290 — APPROVED** [review]
Just a few questions / comment

File: conftest.py (hunk 2 above)
Comment (mlvernay) [reply conftest.py:124]: Yes, your best friend.
